R/Task.R

```diff
@@ -113,13 +107,13 @@ NULL
 #'   weights
 #' @param blocking [\code{numeric}\cr
 #'   task data blocking
-#' @param spatial [\code{logical(1)}]\cr
-#'   whether data is spatial
+#' @param coordinates [\code{logical(1)}]\cr
+#'   whether spatial coordinates have been provided
```

Comment:
This isn't a flag now, it's the actual coordinates, isn't it?

Reply:
Updated it in the public part but forgot it in the internal part. Why are there two docs actually?